### PR TITLE
Added pre-commit hooks for black and flake8

### PR DESCRIPTION
**Pull request by Yttrium-32**

### CODE REVIEW

**Review by Anv3sh — COMMENTED** [review]
don't push `poetry.lock`

**Comment by Yttrium-32** [comment]
Ok right, should I just add it to `.gitignore` then?

**Review by Anv3sh — CHANGES_REQUESTED** [review]
also why added .flake8?

**Comment by Yttrium-32** [comment]
It should make it easier to configure flake8. I was planning on adding an `extend-ignore` section for E501 and F401.

**Comment by Anv3sh** [comment]
> It should make it easier to configure flake8. I was planning on adding an `extend-ignore` section for E501 and F401.

well do it later lets just stick to default for now.

File: poetry.lock
Comment (Anv3sh) [inline poetry.lock:1]: don't delete potery lock jut remove your changes from commit.

**Comment by Yttrium-32** [comment]
You can merge this now if you'd like, I've removed `poetry.lock` and `.flake8` from this pull request.

**Review by Anv3sh — APPROVED** [review]

**Pull request merged**